docs/src/api/class-pageassertions.md

```diff
@@ -323,17 +323,18 @@ expect(page).to_have_url(re.compile(".*checkout"))
 await Expect(Page).ToHaveURLAsync(new Regex(".*checkout"));
 ```
 
-### param: PageAssertions.toHaveURL.urlOrRegExp
+### param: PageAssertions.toHaveURL.urlRegExOrPredicate
```

Comment:
Nit: I think we just use `url` in routes. Unrelated, we use RegExp (JS) or Regex (the rest of the world).